src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -44,7 +44,7 @@ Author: Daniel Kroening, kroening@kroening.com
 void java_bytecode_languaget::get_language_options(const cmdlinet &cmd)
 {
   assume_inputs_non_null=cmd.isset("java-assume-inputs-non-null");
-  java_class_loader.use_core_models=!cmd.isset("no-core-models");
+  java_class_loader.set_use_core_models(!cmd.isset("no-core-models"));
```

Comment:
:+1: but would have been better in a separate commit for future reference

Reply:
Was only necessary because we need to clear a cache now when this is set so specific to this change.